Library/Menu.h

```diff
@@ -6,7 +6,10 @@ class Menu
 public:
 	Menu();
 	void Introduction();
```

Comment:
Menu class contains only one method. do we really need it? maybe we should put some logic into it?